docs/README.md

```diff
@@ -16,6 +16,8 @@ JavaScript and TypeScript [finite state machines](https://en.wikipedia.org/wiki/
 
 New to state machines and statecharts? [Read our introduction](/guides/introduction-to-state-machines-and-statecharts/).
 
+[Download our VSCode extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode).
```

Comment:
do we have an organization account setup so we could publish this in the Stately namespace?